src/main/java/org/molgenis/vcf/decisiontree/runner/DecisionTreeFactoryImpl.java

```diff
@@ -46,11 +52,32 @@
   public DecisionTree map(VcfMetadata vcfMetadata, Settings settings) {
     ConfigDecisionTree configDecisionTree = settings.getConfigDecisionTree();
     Map<String, Label> labelMap = mapLabels(configDecisionTree.getLabels());
-    Map<String, Node> nodeMap = mapNodes(vcfMetadata, configDecisionTree.getNodes(), labelMap);
+    Map<String, List<String>> filesMap = mapFiles(configDecisionTree.getFiles());
```

Comment:
`Set<String>` instead of `List<String>` to allow for performant 'in' queries (e.g. gene panel use case).
In this case the validator should check that `file:` values can't be used with `==` queries,